src/main/scala/cognite/spark/v1/DataModelInstancesRelation.scala

```diff
@@ -174,6 +230,20 @@ object DataModelInstanceRelation {
   private def unknownPropertyType(a: Any) = s"Unknown property type $a."
 
   //scalastyle:off cyclomatic.complexity
+  private def parseValue(value: Any): Json = value match {
+    case x: Double => jsonFromDouble(x)
+    case x: Int => jsonFromDouble(x.toDouble)
+    case x: Float => jsonFromDouble(x.toDouble)
+    case x: Long => jsonFromDouble(x.toDouble)
+    case x: java.math.BigDecimal => jsonFromDouble(x.doubleValue)
+    case x: java.math.BigInteger => jsonFromDouble(x.doubleValue)
+    case x: String => Json.fromString(x)
+    case x: Boolean => Json.fromBoolean(x)
+    case x: Array[Any] =>
+      Json.fromValues(x.map(parseValue))
+    case null => Json.Null // scalastyle:off null
```

Comment:
since this is only used for processing filters, it feels tempting to just `parseJson(value.toString)` :joy:  (after checking for arrays and nulls)

Reply:
where is `parseJson` function 😠

Reply:
show me pls

Reply:
I think it's `io.circe.parser.parse(x).right.get`